src/client/pages/OfferNew/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { AnimateSharedLayout } from 'framer-motion'
+import styled from '@emotion/styled'
+import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { Tab } from './Tab'
+
+const TabList = styled.div`
+  box-shadow: 0px 2px 0px ${colorsV3.gray300};
+  display: flex;
+  margin-bottom: 1.25rem;
+  margin-right: 1rem;
+  margin-top: 1rem;
+  overflow: hidden;
+  max-width: 100%;
+
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    margin-bottom: 2.5rem;
+    margin-top: 1rem;
+    overflow: visible;
+  }
+`
+
+type Props = {
+  insurances: {
+    id: string
+    name: string
+    content?: string
```

Comment:
Is this representing what to display when the tab is selected?

I think we need something more flexible like `React.ReactNode`, right?
Also, should it be optional?